src/components/organisms/socialLoginModalContent.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next';
+import { AiOutlineClose } from 'react-icons/ai';
+import SocialLoginButton from '../atoms/socialLoginButton';
+
+interface SocialLoginModalContentProps {
+  onCloseModalClick: () => {};
```

Comment:
() => {}; 는 함수에서의 블럭으로 간주되는 것이 아니라, {} (빈 객체) 를 반환한다. 라는 의미로 해석됩니다.

```suggestion
  onCloseModalClick: () => void;
```
또는
```suggestion
  onCloseModalClick: React.MouseEventHandler<HTMLButtonElement>;
```
로 변경하는 것이 안전해 보입니다.

Reply:
윗 예시로 수정했습니다 (아래 예시는 수정시 에러가 발생합니다)
![bandicam 2023-10-08 21-37-12-546](https://github.com/Step3-kakao-tech-campus/Team4_FE/assets/87762581/84239e85-850a-4954-86c7-39b030c0b05e)